lib/adapters/xhr.js

```diff
@@ -27,6 +27,10 @@ module.exports = function xhrAdapter(config) {
       }
     }
 
+    if (utils.isFormData(requestData) && utils.isStandardBrowserEnv()) {
+      delete requestHeaders['Content-Type']; // Let the browser set it
```

Comment:
Shouldn't this be done [further down](https://github.com/axios/axios/blob/6ee101b4b5074cf5f0d7f4d007ec181827635f01/lib/adapters/xhr.js#L158) in order to use normalised headers?

Using this here would only catch the specific casing `Content-Type` and not `content-type`